README.md

```diff
@@ -19,7 +19,7 @@ React Native wrapper for the [Spokestack](https://spokestack.io) speech activity
 `$ npm install react-native-spokestack --save`
 
 - _React Native_: 0.60.0+
-- _Android_: Android SDK 26+
+- _Android_: Android SDK 24+
```

Comment:
We're sure this is still the case? Is it an RN thing?

Reply:
That the minimum version is decreasing? It was artificially inflated before.

Reply:
Yeah; I was wondering whether it might still be inflated—it's 8 in the base library and 21 in I think all of the apps we've made.